packages/typescriptlang-org/src/copy/ja/nav.ts

```diff
@@ -0,0 +1,28 @@
+import { navCopy as enNavCopy } from "../en/nav"
+
+export const navCopy = {
+  ...enNavCopy,
+  skip_to_content: "スキップしてメインコンテンツに進む",
+  nav_documentation: "ドキュメント",
+  nav_documentation_short: "ドキュメント",
+  nav_download: "ダウンロード",
+  nav_community: "コミュニティ",
+  nav_playground: "プレイグラウンド",
+  nav_playground_short: "プレイグラウンド",
+  nav_handbook: "ハンドブック",
+  nav_tools: "ツール",
+  nav_search_placeholder: "検索",
+  nav_search_aria: "TypeScript サイト内を検索",
+  nav_beta: "Search the TypeScript site",
+  nav_beta_notification:
```

Comment:
@orta

It looks we no longer used both `nav_beta` and `nav_beta_notification` on the `TopNav`.
Can I remove them ?

https://github.com/microsoft/TypeScript-Website/blob/v2/packages/typescriptlang-org/src/components/layout/TopNav.tsx#L101-L103

Reply:
Yes!

Reply:
@orta
Thank you :bow: I've opened a PR to remove them. ( #898 ).